src/screens/SendSheet.js

```diff
@@ -608,7 +608,7 @@ export default function SendSheet(props) {
     let label = 'Enter an Amount';
 
     let nativeToken = 'ETH';
-    if (network === networkTypes.polygon) {
+    if (currentNetwork === networkTypes.polygon) {
```

Comment:
nice! should we be doing the same thing on line 465?

Reply:
spooky, just changed it. nice catch